airbyte-integrations/connectors/source-datadog/unit_tests/test_streams.py

```diff
@@ -43,6 +43,7 @@ def test_task_stream(requests_mock, stream, config, mock_responses):
 @patch.multiple(DatadogStream, __abstractmethods__=set())
 def test_next_page_token(config):
     stream = DatadogStream(
+        site=config["site"],
```

Comment:
@bartveenstra could you add a unit test which confirms that the outgoing URL is what we expect when parametrizing using this `site` ?